### PR TITLE
tensorflowonspark: new recipe

### PR DESCRIPTION
**Pull request by afloren**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by afloren** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/tensorflowonspark/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4d7a9bd1784112518e1bd49f02db16cb5d8a32c473e6114d89c2f3304da14db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (chrisburr) [inline recipes/tensorflowonspark/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/tensorflowonspark/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - tensorflowonspark
+
+about:
+  home: "https://github.com/yahoo/TensorFlowOnSpark"
+  license: Apache Software
```
Comment (chrisburr) [inline recipes/tensorflowonspark/meta.yaml]: We prefer SPDX identifiers:
```suggestion
  license: Apache-2.0
```

**Comment by afloren** [comment]
@chrisburr ready for review

**Comment by chrisburr** [comment]
LGTM! Thanks for the recipe and welcome to conda-forge! 🍰

**Pull request merged**